gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePlugin.java

```diff
@@ -623,16 +624,11 @@ static void applyDependencyForIdeTasks(Project project, Task compileConjure) {
 
             // module.getSourceDirs / getGeneratedSourceDirs could be an immutable set, so defensively copy
```

Comment:
This is funny, as we say this this but then make sure it's an immutable set to mess with anyone else who wants to do stuff :p